src/battle_util.c

```diff
@@ -7127,6 +7233,10 @@ static u32 CalcDefenseStat(u16 move, u8 battlerAtk, u8 battlerDef, u8 moveType,
         if (gBattleMons[battlerDef].species == SPECIES_CHERRIM && WEATHER_HAS_EFFECT && gBattleWeather & WEATHER_SUN_ANY && !usesDefStat)
             MulModifier(&modifier, UQ_4_12(1.5));
         break;
+    case ABILITY_PUNK_ROCK:
+        if ((gBattleMoves[move].flags & FLAG_SOUND) && !IS_MOVE_STATUS(move))
```

Comment:
I think the check for being a Status move is redundant because only damaging attacks will have Damage Calc commands in their move effect script.

Reply:
That makes sense to me. I'll remove it :eyes: